README.md

```diff
@@ -257,6 +257,7 @@ StreamChat.getOnlineStatus().observe(...);
 - Threads
 - Slash commands
 - Offline support
+- [Markdown messages formatting](https://github.com/noties/Markwon)
```

Comment:
we should have a section about what is supported in terms of Markdown since we have our own flavor

Reply:
True. Updated readme with used plugins.